haddock3.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#===============================================================================
+
+HADDOCK3_DIR="$HOME/repos/haddock3"
+
+# shellcheck source=/dev/null
+source "$HADDOCK3_DIR/venv/bin/activate" || exit
+
```

Comment:
here I would add the commands for activating haddock3 using conda, something like the following

```suggestion
# if your haddock3 installation uses conda
# conda init
# source $HOME/anaconda3/bin/activate
# source activate haddock3
```

Reply:
Indeed, thanks!